experimental/single_number_formatter/src/provider.rs

```diff
@@ -31,38 +31,48 @@ use zerovec::{VarZeroVec, ZeroMap};
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[yoke(prove_covariance_manually)]
 pub struct CurrencyEssentialsV1<'data> {
+    /// Maps from currency iso code to currency patterns
+    /// which points to which pattern to use and the place holder index.
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub currency_patterns_map: ZeroMap<'data, UnvalidatedTinyAsciiStr<3>, CurrencyPatterns>,
 
+    /// Represents the standard pattern.
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub standard: Cow<'data, str>,
 
+    /// Represents the standard alpha_next_to_number pattern.
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub standard_alpha_next_to_number: Cow<'data, str>,
 
+    /// Contains all the place holders.
```

Comment:
Good docs should do more than re-stating the thing it is documenting. I already know that this field contains "place holders" because that's what the field is named. But what is a place holder? Is the order of the place holders in the vector of importance? And so forth.